account_cryptocurrency/data/res_currency_move_sequence.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+    <record id="res_currency_move_sequence" model="ir.sequence">
+        <field name="name">Currency Move Sequence</field>
+        <field name="code">res.currency.move</field>
+        <field name="prefix">CURR.MV/%(range_year)s/</field>
```

Comment:
I personally think that sequence should be short and without "/" like CM180001 but non blocking